From 0fc1e72735c123b3491faca0cefce1fe288bee79 Mon Sep 17 00:00:00 2001
From: Alex Leibovich <alexl@marvell.com>
Date: Tue, 26 Feb 2019 13:52:19 +0200
Subject: [PATCH 1036/1200] ci: added ddr32 relevant builds

Added u-boot build for 80x0 and 9130
with support in ddr32

Change-Id: Ic75e1f3c6c03361d53bbda1b7855cc728e350475
Signed-off-by: Alex Leibovich <alexl@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/4840
Reviewed-by: Lior Levin <Lior.Levin@cavium.com>
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
Tested-by: sa_ip-sw-jenkins
---
 scripts/ci/compile.sh           | 2 ++
 scripts/ci/supported_builds.txt | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/scripts/ci/compile.sh b/scripts/ci/compile.sh
index 60257c3859..be604e0d7f 100755
--- a/scripts/ci/compile.sh
+++ b/scripts/ci/compile.sh
@@ -113,6 +113,7 @@ case $build_name in
 	uboot_2018_a80x0_H ) device_tree="armada-8040-db-H"; ;;
 	uboot_2018_a80x0_kr ) device_tree="armada-8040-db"; ;;
 	uboot_2018_a80x0_pm ) device_tree="armada-8040-db"; ;;
+        uboot_2018_a80x0_ddr32 ) device_tree="armada-8040-db"; ;;
 	uboot_2018_a80x0_mcbin_single_shot* )
 		device_tree="armada-8040-mcbin-single-shot"; ;;
 	uboot_2018_a80x0_mcbin* ) device_tree="armada-8040-mcbin"; ;;
@@ -121,6 +122,7 @@ case $build_name in
 	uboot_2018_a3900_A ) device_tree="armada-3900-vd-A"; ;;
 	uboot_2018_a3900_B ) device_tree="armada-3900-vd-B"; ;;
 	uboot_2018_cn9130_A ) device_tree="cn9130-db-A"; ;;
+        uboot_2018_cn9130_ddr32_A ) device_tree="cn9130-db-A"; ;;
 	uboot_2018_cn9131_A ) device_tree="cn9131-db-A"; ;;
 	uboot_2018_cn9132_A ) device_tree="cn9132-db-A"; ;;
 
diff --git a/scripts/ci/supported_builds.txt b/scripts/ci/supported_builds.txt
index 1516666b57..2fe9f32e31 100644
--- a/scripts/ci/supported_builds.txt
+++ b/scripts/ci/supported_builds.txt
@@ -29,10 +29,12 @@ uboot_2018_a80x0_mcbin_single_shot
 uboot_2018_a80x0_mcbin_single_shot_cve_2018_3639
 uboot_2018_a80x0_mcbin_single_shot_pm
 uboot_2018_a80x0_ucpe
+uboot_2018_a80x0_ddr32
 
 uboot_2018_a3900_A
 uboot_2018_a3900_B
 uboot_2018_cn9130_A
+uboot_2018_cn9130_ddr32_A
 uboot_2018_cn9131_A
 uboot_2018_cn9132_A
 
-- 
2.22.0

